### PR TITLE
Add GraphQL docs

### PR DESCRIPTION
**Pull request by tomekpapiernik**
**Description**

Changes proposed in this pull request:

- Add user-oriented documentation describing Kyma GraphQL implementation. 
- Change types and numbering of "Security" documents to reflect their content (Details -> Tutorials)
- Minor bugfixes and tweaks

**Related issue(s)**
#277

### CODE REVIEW

File: docs/console/docs/03-02-ui-api-layer.md [hunk 1]
```diff
@@ -5,13 +5,15 @@ type: Details
 
 The UI API Layer is a backend service which provides an API for all views of the Console UI. This service consumes the Kubernetes API and exposes a simplified GraphQL API to allow frontends to perform Kubernetes resource operations.
 
+> **NOTE:** Read [this](/docs/root/components/security#details-graphql) security document for more information about the Kyma GraphQL implementation.
```
Comment (bszwarc) [inline docs/console/docs/03-02-ui-api-layer.md]:
```suggestion
> **NOTE:** Read [this](/docs/components/security#details-graphql) security document for more information about the Kyma GraphQL implementation.
```
remove `root`. The document type for security is `components`. `root` is the type for docs in `kyma/docs/kyma`.

File: docs/security/docs/03-02-graphql.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: GraphQL
+type: Details
+---
+
+Kyma uses a custom [GraphQL](https://graphql.org/) implementation in the UI API Layer and deploys an RBAC-based logic to control access to the GraphQL endpoint. All calls to the GraphQL endpoint require a valid Kyma token for authentication.
```
Comment (bszwarc) [inline docs/security/docs/03-02-graphql.md]:
```suggestion
Kyma uses a custom [GraphQL](https://graphql.org/) implementation in the UI API Layer and deploys an RBAC-based logic to control the access to the GraphQL endpoint. All calls to the GraphQL endpoint require a valid Kyma token for authentication.
```

File: docs/security/docs/03-02-graphql.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+The authorization in GraphQL uses RBAC, which means that:
+  - All of the Roles, RoleBindings, ClusterRoles and CluserRoleBindings that you create and assign are effective and give the same permissions when the users interact with the cluster resources both through the CLI and the GraphQL endpoints.
```
Comment (bszwarc) [inline docs/security/docs/03-02-graphql.md]:
```suggestion
  - All of the Roles, RoleBindings, ClusterRoles and CluserRoleBindings that you create and assign are effective and give the same permissions when users interact with the cluster resources both through the CLI and the GraphQL endpoints.
```

File: docs/security/docs/03-02-graphql.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+| **query** | get (for a single resource), list (for multiple resources) |
+| **mutation** | create, delete |
+| **subscription** | watch |
+
+> **NOTE:** Due to the nature of Kubernetes, you can secure specific resources specified by their name only for queries and mutations. Subscriptions work only with entire resource groups, such as kinds, and therefore don't allow for such level of granularity.
+
+## Available GraphQL actions
+
+To access cluster resources through GraphQL, an action securing given resource must be defined and implemented in the cluster.
+See the [GraphQL schema](https://github.com/kyma-project/kyma/blob/master/components/ui-api-layer/internal/gqlschema/schema.graphql) file to see the list of actions implemented in every Kyma cluster by default.
```
Comment (bszwarc) [inline docs/security/docs/03-02-graphql.md]:
```suggestion
See the [GraphQL schema](https://github.com/kyma-project/kyma/blob/master/components/ui-api-layer/internal/gqlschema/schema.graphql) file for the list of actions implemented in every Kyma cluster by default.
```
I would get rid of one `see` :)

Comment (tomekpapiernik) [reply docs/security/docs/03-02-graphql.md]: :D sure!

**Pull request merged**